Linux build of the master branch fails on Azure Pipelines

```
[364/895] compile ../../../../../../../skia-bindings/skia/src/gpu/gl/glx/GrGLMakeNativeInterface_glx.cpp
FAILED: obj/src/gpu/gl/glx/gpu.GrGLMakeNativeInterface_glx.o 
clang++ -MMD -MF obj/src/gpu/gl/glx/gpu.GrGLMakeNativeInterface_glx.o.d -DNDEBUG -DSK_R32_SHIFT=16 -DSK_GAMMA_APPLY_TO_A8 -DSKIA_IMPLEMENTATION=1 -DSK_GL -I../../../../../../../skia-bindings/skia -fstrict-aliasing -fPIC -O3 -fdata-sections -ffunction-sections -Wno-unused-parameter -std=c++14 -fno-exceptions -fno-rtti -c ../../../../../../../skia-bindings/skia/src/gpu/gl/glx/GrGLMakeNativeInterface_glx.cpp -o obj/src/gpu/gl/glx/gpu.GrGLMakeNativeInterface_glx.o
../../../../../../../skia-bindings/skia/src/gpu/gl/glx/GrGLMakeNativeInterface_glx.cpp:15:10: fatal error: 'GL/glx.h' file not found
#include <GL/glx.h>
         ^~~~~~~~~~
1 error generated.
```

--- a/azure-pipelines-template.yml
+++ b/azure-pipelines-template.yml
@@ -85,6 +85,13 @@ jobs:
       cargo -V
     displayName: Update Rust, Install Clippy and Rustfmt and print Rust & Cargo Versions
 
+  - ${{ if eq(parameters.platform, 'Linux') }}:
+    # Linux Ubuntu 16.04 (Gl/glx.h and libgl)
+    - bash: |
+        set -e
+        sudo apt-get install mesa-common-dev libgl1-mesa-dev libgl1
+      displayName: Install mesa-common-dev libgl1-mesa-dev and libgl1
+
   - ${{ if ne(parameters.platform, 'Windows') }}:
     # Linux and macOS.
     - script: |

